cloud/jenkins/pxc_containers_docker_build.groovy

```diff
@@ -13,6 +13,8 @@ void build(String IMAGE_PREFIX){
             docker build --no-cache --squash -t perconalab/percona-xtradb-cluster-operator:master-${IMAGE_PREFIX} -f pxc-57-backup/Dockerfile pxc-57-backup
         elif [ ${IMAGE_PREFIX} = pxc8.0-backup ]; then
             docker build --no-cache --squash -t perconalab/percona-xtradb-cluster-operator:master-${IMAGE_PREFIX} -f pxc-80-backup/Dockerfile pxc-80-backup
+        elif [ ${IMAGE_PREFIX} = haproxy2.1 ]; then
```

Comment:
we don't support multiple haproxy versions in one operator version, please lets simplify `s/haproxy2.1/haproxy/`

Reply:
ok